projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.spec.ts

```diff
@@ -6,20 +6,16 @@ import { PolicyService } from '../../../core/my-account/facade';
 import { PremiumCalendarComponent } from './premium-calendar.component';
 import createSpy = jasmine.createSpy;
 import { of } from 'rxjs';
-
```

Comment:
Put empty line between imports and actual code and between test methods. Now it is all glued together :)